salt/profiles/postgresql.sls

```diff
@@ -37,3 +37,37 @@ profiles-db:
         {% endif %}
         - require:
             - profiles-db-user
+
+    cmd.run:
```

Comment:
Why's this needed?

Reply:
Is the `db_user` above not always the owner?

Reply:
`pillar.profiles.db.username` has been set as the owner of the database, but `pillar.elife.db_root.username` is still the owner of the schema; the schema is inside of the database so it should be made owned by the same user as the database.
Salt provides a `postgres_schema.exists` state, but it doesn't change the ownership (bug), so a manual command is needed.

Reply:
👍